wee_slack.py

```diff
@@ -617,6 +617,10 @@ def buffer_prnt(self, user='unknown_user', message='no message', time=0):
             set_read_marker = True
         elif message.find(self.server.nick.encode('utf-8')) > -1:
             tags = "notify_highlight"
+        elif slack_highlight_words and message.find("!channel".encode('utf-8')) > -1:
```

Comment:
slack_highlight_words isn't marked as a global (and we should avoid globals anyway). Can you just grab the setting from weechat near the top of the function?